packages/cli/junit.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is autogenerated but not excluded from .gitignore, should it be kept?

Reply:
@danstarns This is a recent new file, looks like we forgot to add to .gitignore, could you remove it from this PR and add it in .gitignore?

Reply:
Was introduced in https://github.com/prisma/prisma/pull/12044

Reply:
Sure!